PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -146,14 +146,19 @@
             style.paddingLeft = Int(insets.left)
         }
 
-        private func createBasicWireframe(_ window: UIView) -> RRWireframe {
+        private func createBasicWireframe(_ view: UIView) -> RRWireframe {
```

Comment:
the first time `toWireframe` gets called, we always pass the `window`, so maybe this method should have 2 params, the view and the window.
`view.window` returns nullable which is likely never the case, but if it does, the positioning will be broken again, passing the `view` and `window`, we know both are never nullable, similar to `findMaskableWidgets`